pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/OncHistoryDetail.java

```diff
@@ -322,6 +321,46 @@ public static OncHistoryDetail getOncHistoryDetail(@NonNull String oncHistoryDet
         return (OncHistoryDetail) results.resultValue;
     }
 
+    public static OncHistoryDetail getRandomOncHistoryDetail(@NonNull String oncHistoryDetailId, String realm) {
```

Comment:
There is `OncHistoryDetailDao` and its implementation `OncHistoryDetailDaoImpl` and put this method there without static, and please for future, any method which is responsible to fetch data from database use `Dao` classes.

Reply:
I agree with @gmakharat. It would be great if we follow `Dao` pattern for multiple reasons, chief among them: having common language and shared reasoning about code.

Reply:
I agree with @gmakharat. It would be beneficial for the team as a whole if we follow some of the best practices `Dao` pattern being one of them. It will make easier for all of us to understand the code for multiple reasons, chief among them: having common language and shared reasoning.